chemicaltoolbox/autodock_vina/prepare_receptor/prepare_receptor.xml

```diff
@@ -0,0 +1,92 @@
+<tool id="prepare_receptor" name="Prepare receptor" version="0.1.0">
+    <description>- Tool to prepare receptor for Autodock Vina</description>
+    <requirements>
+        <requirement type="package" version="1.5.6">mgltools</requirement>
+    </requirements>
+    <stdio>
+        <exit_code range="1" />
+    </stdio>
+    <command><![CDATA[
+        prepare_receptor4.py -r "$receptor" -o "$file_output" -v -U nphs_lps -A hydrogens
+    ]]></command>
+    <inputs>
+        <param type="data" name="receptor" format="pdb" label="Receptor: " help="Select a pdb file." />
+    </inputs>
+    <outputs>
+        <data name="file_output" format="pdbqt" label="#echo os.path.splitext (str($receptor.name))[0]#.pdbqt" />
```

Comment:
This somehow assumes that you input is labeld ****.pdb isn't it?
I don't think this will be often the case. I would take the standard Galaxy label.